### PR TITLE
Highlight hover

### PR DESCRIPTION
**Pull request by vachan-maker**
My attempt to add highlight hover to the user info panel

### CODE REVIEW

**Comment by vachan-maker** [comment]
I think this is completely wrong

File: src/UserInfoWidget.cpp [hunk 1]
```diff
@@ -216,4 +216,17 @@ UserInfoWidget::paintEvent(QPaintEvent *event)
         opt.init(this);
         QPainter p(this);
         style()->drawPrimitive(QStyle::PE_Widget, &opt, &p, this);
+  
+        Painter p(this);
```
Comment (deepbluev7) [inline src/UserInfoWidget.cpp:220]: You forgot to include this, so it fails to compile.

File: src/UserInfoWidget.cpp [hunk 2]
```diff
@@ -216,4 +216,17 @@ UserInfoWidget::paintEvent(QPaintEvent *event)
...
+        PainterHighQualityEnabler hq(p);
+
+        if (isPressed_)
+                p.fillRect(rect(), highlightedBackgroundColor_);
+        else if (isDisabled_)
+                p.fillRect(rect(), disabledBackgroundColor_);
+        else if (underMouse())
```
Comment (deepbluev7) [inline src/UserInfoWidget.cpp:227]: I think this will always return false, since setMouseTracking wasn't called with `true`.

**Review by deepbluev7 — COMMENTED** [review]
The idea looks good to me, I think this still needs some work to actually work, but it seems to be on the right track. Thanks! :3

**Comment by vachan-maker** [comment]
@deepbluev7 I copied and pasted the code without any understanding, so I may not be able to finish this.

File: src/UserInfoWidget.cpp (hunk 1 above)
Comment (vachan-maker) [reply src/UserInfoWidget.cpp:221]: Included #include "ui/Painter.h

File: src/UserInfoWidget.cpp (hunk 2 above)
Comment (vachan-maker) [reply src/UserInfoWidget.cpp:229]: setMouseTracking(true); done

**Pull request closed without merging**